src/data/games.json

```diff
@@ -22,5 +22,9 @@
     {
         "title": "Celeste",
         "platforms": [ "PC", "Xbox one", "Nintendo Switch" ]    
+    },
+    {
+        "title": "Getting Over It with Bennett Foddy",
```

Comment:
Are we sure that this is spelled correctly? Just double checking, thanks. :)

Reply:
Yes It is